### PR TITLE
Eslint for /cdk

### PR DESCRIPTION
**Pull request by synnovsh**
Adds linting with eslint with eslint-config-prettier as suggested by [Prettier](https://prettier.io/docs/en/integrating-with-linters.html).

### CODE REVIEW

File: cdk/backend/function/createUserformBatch/index.js [hunk 1]
```diff
@@ -371,42 +369,58 @@ async function insertQuestionAnswers(answerArray, userformId) {
...
-          orgAdmins: `${orgId}0admin`,
-          orgGroupLeaders: `${orgId}0groupLeader`,
-          __typename: 'UserForm',
-        },
-      })
-      .promise()
-    console.log('Create userform result: ', {
-      userform: userformId,
-      result: result,
+  const userformId = uuidv4()
```
Comment (synnovsh) [inline cdk/backend/function/createUserformBatch/index.js:372]: removed try/catch block

File: cdk/backend/function/createUserformBatch/index.js [hunk 2]
```diff
@@ -371,42 +369,58 @@ async function insertQuestionAnswers(answerArray, userformId) {
...
+    result: result,
+  })
+  return userformId
 }
 
 //Returns the X newest userforms by ownerId
 async function getUserformByOwnerWithLimit(formDefID = undefined, limit = 1) {
-  try {
-    let result = await docClient
+  let result = await docClient
```
Comment (synnovsh) [inline cdk/backend/function/createUserformBatch/index.js:397]: removed try/catch block

File: cdk/backend/function/createUserformBatch/index.js [hunk 3]
```diff
@@ -415,62 +429,34 @@ async function getUserformByOwnerWithLimit(formDefID = undefined, limit = 1) {
...
-        },
-        ConsistentRead: false,
-      })
-      .promise()
-    console.log('QuestionAnswer query result: ', result)
-    return result.Items
-  } catch (err) {
-    throw err
-  }
+  let result = await docClient
```
Comment (synnovsh) [inline cdk/backend/function/createUserformBatch/index.js:447]: removed try/catch block

File: cdk/backend/presignup/custom.js [hunk 4]
```diff
@@ -17,19 +17,8 @@ const OrganizationConstants = {
   IdentifierAttribute: 'identifierAttribute',
 }
 
-const isDeveloperLogin = event => {
```
Comment (synnovsh) [inline cdk/backend/presignup/custom.js:20]: Functions not used

**Review by krisstam — APPROVED** [review]
Jeg er litt usikker på de try/catch blokkene vi hadde i createUserFromBatch, men ellers ser det ok ut. Tror ikke de burde gjøre en forskjell fra eller til

**Review by andersrebner — APPROVED** [review]

**Pull request merged**